src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -62,12 +64,157 @@ public void TestRootGetProcessById()
         }
 
         [Fact]
-        public void TestUseShellExecute_Unix_Succeeds()
+        public void ProcessStart_TryExitCommandAsFileName_ThrowsWin32Exception()
+        {
+            Win32Exception e = Assert.Throws<Win32Exception>(() => Process.Start(new ProcessStartInfo { UseShellExecute = false, FileName = "exit", Arguments = "42" }));
+        }
+
+        [Fact]
```

Comment:
this test and the one below, do they match Windows behavior? If so they shoud lbe in ProcessTests.cs (ie., not Unix specific file). This will help make certain the behavior stays the same betwen Unix and windows.

Reply:
and if they don't match Windows behavior, hopefully we thought that through, and a comment here explaining why we chose to diverge would be good